### PR TITLE
Add conditional execution docs

### PR DESCRIPTION
**Pull request by antepusic**
### Description

Documentation for the conditional execution query module ([link](https://github.com/memgraph/mage/pull/203)).

MAGE changelog message:
> The new conditional execution module lets you define conditions not expressible in Cypher and and use them to control query execution.

### Pull request type

- [x] New documentation page

### Related PRs and issues

PR this doc page is related to: [MAGE PR#203](https://github.com/memgraph/mage/pull/203)

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

File: mage/query-modules/python/conditional-execution.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+* constraint creation/deletion
+* changing the isolation level globally
+* setting the storage mode
+
+:::
+
+### `case(conditionals, else_query, params)`
+
+Given a list of condition-query pairs, `do.case` executes the query associated
+with the first condition evaluating to `true` (or the `else query` if none are
```
Comment (vpavicic) [inline mage/query-modules/python/conditional-execution.md:53]:
```suggestion
with the first condition evaluating to `true` (or the `else query` if none are
```
This sounds strange... "the first condition evaluating TO `true`
What do you mean by this? "associated with the first condition that is/results in `true` " can we just say this?

File: mage/query-modules/python/conditional-execution.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Parameters are prefixed with `$` like `$param_name`. For examples, see
+[here](https://memgraph.com/docs/cypher-manual/other-features#parameters).
+
+#### Input:
+
+* `conditionals: List[Any]` ➡ Variable-length list of condition-query pairs
+  structured as `[condition, query, condition, query, …​]`. Conditions are
+  `boolean` and queries are `string`.
+* `else_query: string (default = "")` ➡ The query to be executed if no
+  condition evaluates to true.
```
Comment (vpavicic) [inline mage/query-modules/python/conditional-execution.md:65]:
```suggestion
  condition evaluates to `true`.
```

File: mage/query-modules/python/conditional-execution.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+Parameters are prefixed with `$` like `$param_name`. For examples, see
+[here](https://memgraph.com/docs/cypher-manual/other-features#parameters).
+
+#### Input:
+
+* `condition: boolean` ➡ Determines what query to execute.
+* `if_query: string` ➡ The query to be executed if the condition is satisfied.
+* `else_query: string (default = "")` ➡ The query to be executed if the
+  condition isn’t satisfied.
+* `params: Map (default = NULL)` ➡ If `if_query` or `else_query` parameterized,
```
Comment (vpavicic) [inline mage/query-modules/python/conditional-execution.md:101]:
```suggestion
* `params: Map (default = NULL)` ➡ If `if_query` or `else_query` are parameterized,
```

File: mage/query-modules/python/conditional-execution.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+   provide a `{param_name: param_value}` map to be applied.
```
Comment (vpavicic) [inline mage/query-modules/python/conditional-execution.md:102]:
```suggestion
   provide a `{param_name: param_value}` map to be applied.
```

that will be applied?

Comment (antepusic) [reply mage/query-modules/python/conditional-execution.md:102]: Both are valid constructions and unambiguous, but feel free to make this change if you think it’s more understandable :)

**Pull request merged**